build_scripts/run_pylint.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Copyright (C) 2023 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# AQuaSurF Software in commercial settings.
+#
+# END COPYRIGHT
+
+RCFILE=build_scripts/pylintrc
+UP_TO_SNUFF_DIRS="aquasurf"
+
+dirs=$1
+if [ "x${dirs}" == "x" ]
+then
+    dirs=${UP_TO_SNUFF_DIRS}
+fi
+
+retval=0
+for dir in ${dirs}
+do
+    echo "Running pylint on directory '${dir}':"
+    find "${dir}" -iname "*.py" | \
+        xargs pylint --load-plugins=pylint_protobuf --rcfile=${RCFILE}
```

Comment:
Can't help but notice that pylint_protobuf is not in your requirements-build.txt at all, so maybe get rid of the --load-plugins arg.

Reply:
Done